apparmor.d/profiles-a-f/acpid

```diff
@@ -20,17 +20,68 @@ profile acpid @{exec_path} flags=(attach_disconnected) {
   /{usr/,}bin/{,ba,da}sh        rix,
   /{usr/,}bin/logger            rix,
 
+  /etc/acpi/powerbtn-acpi-support.sh rPx -> acpid//powerbtn-acpi-support.sh,
+
   /etc/acpi/{,**} r,
   /etc/acpi/handler.sh rix,
 
+  owner @{run}/acpid.socket rw,
+  owner @{run}/acpid.pid    rw,
+
+  owner @{PROC}/@{pids}/fd/ r,
+  owner @{PROC}/@{pids}/loginuid r,
+
   /dev/input/{,**} r,
   /dev/tty rw,
-  /dev/null r,
 
-  owner @{PROC}/@{pid}/fd/ r,
-  owner @{PROC}/@{pid}/loginuid r,
+  include if exists <local/acpid>
+}
 
-  @{run}/acpid.socket rw,
+profile acpid//powerbtn-acpi-support.sh flags=(attach_disconnected) {
+  include <abstractions/base>
+  include <abstractions/wutmp>
 
-  include if exists <local/acpid>
+       capability sys_ptrace,
+  deny capability net_admin,  # ??
+
+  ptrace (read),  # unconfined, tighten later, TODO
+
+  /etc/acpi/powerbtn-acpi-support.sh r,
+
+  /{usr/,}bin/sed          rix,
+  /{usr/,}bin/pgrep        rix,
+  /{usr/,}bin/{,e}grep     rix,
+  /{usr/,}bin/pinky        rix,
+  /{usr/,}bin/{,ba,da}sh   rix,
+  /{usr/,}bin/dbus-send    rix,
+  /{usr/,}bin/systemctl    rix,
+  /{usr/,}sbin/killall5    rix,
```

Comment:
Some distributions do not have sbin directory (eg: arch). So you should use `{s,}bin`.